tests/common/random.rs

```diff
@@ -310,5 +310,25 @@ mod tests {
             random_string.as_bytes().iter().filter(|p| **p == 0).count()
         );
         assert!(random_string.as_bytes().ends_with(&[0]));
+
+        // This test exists to exclude an error (timeout) occurred during system tests on windows
```

Comment:
How does it do this?

Reply:
It does so because this is the exact code from the test in which I used the random generator and the test timed out. But there's more code in the test, so to exclude the possibility that it has something to do with the random string generator, I added these lines from the test here and test them separately.

Reply:
Hmm, ok, I'm not 100% if this is useful, but I think it's a vit better to at least add that explanation to the code here